index.js

```diff
@@ -40,12 +40,39 @@ class ServerlessPythonRequirements {
   };
 
   /**
+   * parse requirements.txt into .requirements.txt, leaving out #no-deploy lines
+   * @return true
+   */
+  parseRequirements() {
+    if (!fse.existsSync(path.join(this.serverless.config.servicePath,
+                                  'requirements.txt'))) {
+      return BbPromise.resolve();
+    }
+
+    this.serverless.cli.log(
+      `Parsing Python requirements.txt`);
+
+    var fs = require('fs');
+    var reqs = fs.readFileSync("requirements.txt").toString().split('\n');
+
+    var newReqs = ''
+    for (var i in reqs) {
+      if(reqs[i].indexOf('#no-deploy') == -1) {
+        newReqs += reqs[i] + '\n'
+      }
+    }
+    fs.writeFileSync(".requirements.txt", newReqs, 'utf8');
```

Comment:
(BTW, just using `mktemp` as an example, you obviously can't write this into `/tmp` and have it be visible.)